packages/wallets/src/components/DatePicker/DatePicker.tsx

```diff
@@ -73,7 +75,7 @@ const DatePicker = ({
                 showMessage
                 type='date'
                 validationSchema={validationSchema}
-                value={selectedDate !== null ? unixToDateString(selectedDate) : ''}
+                value={selectedDate !== null ? unixToDateString(selectedDate) : formValues[name]}
```

Comment:
There was an issue in the Datepicker, where upon typing a date it will get cleared:
Before
https://github.com/binary-com/deriv-app/assets/103016120/088ab647-daee-47ce-9ddd-ebfc88e76f2e
After
https://github.com/binary-com/deriv-app/assets/103016120/f3679309-9d9e-4016-aa9c-8941716b5190